packagelist/debian_12_headless.txt

```diff
@@ -73,10 +83,24 @@ autoconf
 automake
 build-essential
 command-not-found
+dialog
 gdb
 git
+initramfs-tools
+libgpiod-dev
 libiio-dev
+libiio0-doc
 libubootenv-tool
 lsof
 meson
 nfs-common
+sqlite3
+xxd
+
+# Development tools - C/C++
+
```

Comment:
It looks like none are listed under this comment?

Reply:
I had started thinking that the C/C++ development libraries currently under the "Development" heading should have their own heading if the Python development libraries did. I take that back !-)